### ISSUE TITLE
Accessing dataset

### ISSUE THREAD

**Issue by abskjha** [Mingxiao-Li/vln_ce#1]
Hi,
Thanks @Mingxiao-Li for creating the repository, it will help us properly manage the work.
I think many of us are not very familiar with the dataset yet. If we could play with the dataset, it would give more clarity about the challenges. Can you share any link or if you have created any tool/dataloader to access the data.

**Comment by kkelchte** [comment]
Yes indeed, it would be good to be play around with training environments and get a personal human baseline performance. Abisjek, if I remember correctly, you already created a jupyter notebook for this?

**Comment by abskjha** [comment]
@kkelchte Yes, you can find that jupyter notebook in  [notebooks](https://github.com/Mingxiao-Li/vln_ce/tree/master/notebooks).
